src/main/java/duke/task/Task.java

```diff
@@ -0,0 +1,35 @@
+package duke.task;
+
+public abstract class Task {
+    private String name;
+    private boolean isDone;
+
+    private static int taskCount = 0;
+
+    public Task(String name) {
+        this.name = name;
+        isDone = false;
+        taskCount += 1;
+    }
+
+    public void setIsDone() {
+        isDone = true;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public boolean getIsDone() {
+        return isDone;
+    }
+
+    @Override
+    public String toString() {
+        return ("[" + (isDone ? "Y" : "N") + "] " + name);

```

Comment:
Good, changing to Y and N instead of tick and cross is good, then this program will not have the weird '?' bug like many programs has.